app/views/api/v1/search/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.array! organizations, :names, :tag_names, :website_urls, :cities, :countries, :id, :in_business
```

Comment:
Interesting that the json builder is now API specific whereas previously the `as_json` was just a generic json representation declared by the model itself. Will each API have to decide how it wants to represent organizations? While this gives flexibility, it's nice to have a default representation as that will standardize our API responses and it means the API doesn't have to decide each time what an Organization is.

Reply:
Yeah, I see what you mean - it's possible that duplication will creep in here, so something to watch for, but I like this info being in the view layer. In my experience you either quickly discover something to extract or it turns out the duplication is fine.